CONTRIBUTING.md

```diff
@@ -17,7 +17,7 @@ If you want to make changes yourself, follow these steps:
 2. Make your changes
 3. Test your changes
 ```bash
- docker build -t pg-schema-diff-test-runner -f ./build/Dockerfile.test --build-arg POSTGRES_PACKAGE=postgresql{14, 15} .
+ docker build -t pg-schema-diff-test-runner -f ./build/Dockerfile.test .
```

Comment:
The command that was written here would fail because the `Dockerfile.test` file already specifies an `ARG` so I wanted to update this so others are not lead astray.

Reply:
I'm not sure this is true. A The [build arg ](https://docs.docker.com/reference/dockerfile/#arg)specified is the default build arg if one is not specified. 

```
docker build -t pg-schema-diff-test-runner14 -f ./build/Dockerfile.test --build-arg POSTGRES_PACKAGE=postgresql15
```
works for me

Reply:
That command also works for me if I include the current directory arg i.e. `.` when in the root of the repo. The original command still does not work, should I update the doc to the command in your comment?

Reply:
I updated the contributing to be a bit clearer, both commands are confirmed to build.